ipa-core/src/net/test.rs

```diff
@@ -9,46 +9,171 @@
 
 #![allow(clippy::missing_panics_doc)]
 use std::{
-    array,
+    collections::HashSet,
```

Comment:
Consider splitting this file into multiple - it grew too big and Github now hides all the changes for it by default

Reply:
I'm thinking on separating the TestServer stuff